djcelery_email/tasks.py

```diff
@@ -19,7 +19,11 @@
 
 
 @shared_task(**TASK_CONFIG)
-def send_emails(messages, backend_kwargs):
+def send_emails(messages, backend_kwargs={}, **kwargs):
+    # backward compat: handle **kwargs and missing backend_kwargs
+    backend_kwargs = backend_kwargs.copy()  # Avoid modifying mutable default
```

Comment:
I see you tried to mitigate that, but I still prefer to set the default in the arg to `None` then set a fresh default in the function body.

```python
def send_email(messages, backend_kwargs=None, **kwargs):
    final_kwargs = {}
    if backend_kwargs:
        final_kwargs.update(backend_kwargs)
    
    final_kwargs.update(kwargs)
    ...
```